test/configure-jenkins-environment.sh

```diff
@@ -18,6 +18,6 @@ cp env .env
 # .env is used as a configuration file for the rest of the project.
 # Need to choose some values for the automated tests in Jenkins
 sed -i "s/export K8S_VER=/export K8S_VER=1.9.7/g" .env
-sed -i "s/export NEW_K8S_VER=/export NEW_K8S_VER=1.10.4/g" .env
+sed -i "s/export NEW_K8S_VER=/export NEW_K8S_VER=1.10.7/g" .env
 sed -i "s/export GKE_VER=/export GKE_VER=1.9.7-gke.6/g" .env
-sed -i "s/export NEW_GKE_VER=/export NEW_GKE_VER=1.10.2-gke.4/g" .env
+sed -i "s/export NEW_GKE_VER=/export NEW_GKE_VER=1.10.7-gke.6/g" .env
```

Comment:
Can we pull this version with a gcloud command?  Wonder if we should generate the .env file more generically.